update-sdk/README.md

```diff
@@ -0,0 +1,32 @@
+### Overview
+
+A github action to update the SDK packages.
+
+### How to use?
+
+```yml
+- uses: FuelLabs/github-actions/update-sdk@master
+  with:
+    branch: master # In case you want to use a different head branch
+    packages: |
+      @fuels
+      @fuels/react
+      @fuels/connectors
+    npm-tag: ${{ matrix.tag }} # You might use "latest" or anything else, like a matrix strategy
+  env:
+    GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+```
+
+### Inputs
+
+| Name         | Description                                                |
+| ------------ | ---------------------------------------------------------- |
+| npm-tag      | NPM tag (e.g. latest or next)                              |
```

Comment:
We should add the other inputs for repository, branch....

Reply:
https://github.com/FuelLabs/github-actions/pull/26/commits/30378159fa1e43b65545271ec435ecd01570def9